example-password-valid.js

```diff
@@ -0,0 +1,9 @@
+const bcrypt = require('bcrypt');
```

Comment:
rewrite this as test and add it to gulp test series

Reply:
I am not familiar with node js and even less with node js testing.
Though i don't think it makes sense to test the bcrypt library. I think we can trust that their password hashing is testet enough. 
It will be better to test the user authentication in _modules/users.js_ by adding an example user with the tool i have added just now and then try to authenticate the user.